performancecopilot/pcp
# qa and other-packages info updates Changes committed to git@github.com:kmcdonell/pcp.git 20191015

Ken McDonell (7):
      qa/admin/other-packages/manifest: updates for Arch Linux
      qa/check.callback.sample: refine ./870 output cmp
      qa/admin/other-packages/unavailable: 32-bit Fedora 29 changes
      qa/README.vm: update OpenSuSE version availability info
      qa/admin/other-packages/*: CentOS 7.7 updates
      qa/admin/other-packages/*: updates for openSUSE Leap 15.1
      qa/common: fix some logic errors

 qa/README.vm                        |    5 
 qa/admin/other-packages/manifest    |  200 ++++++++++++++++++------------------
 qa/admin/other-packages/unavailable |    7 -
 qa/check.callback.sample            |    7 -
 qa/common                           |    5 
 5 files changed, 113 insertions(+), 111 deletions(-)

Details ...

commit bb3cd75aa60111ca25ed7a60159f46bcc81e8f9a
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Thu Oct 17 09:41:06 2019 +1100

    qa/common: fix some logic errors
    
    Found on first run for a virgin install on openSUSE Leap 15.1,
    although the issues have nothing to do with openSUSE per se.

commit 6df901ea97148bcf68c29817b12fde6118dff2a0
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Thu Oct 17 09:07:34 2019 +1100

    qa/admin/other-packages/*: updates for openSUSE Leap 15.1

commit c4d877f59d1bc29f1f70bcb83ca8aeb0f13cd249
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Wed Oct 16 14:42:05 2019 +1100

    qa/admin/other-packages/*: CentOS 7.7 updates
    
    Similar updates also for RHEL 7.7

commit eaf82f52f202d5e96297e55c30b75cf42146b65d
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Wed Oct 16 11:26:45 2019 +1100

    qa/README.vm: update OpenSuSE version availability info

commit b1078a63d4095409116261b235fa9faad4e5c5a6
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Wed Oct 16 07:34:57 2019 +1100

    qa/admin/other-packages/unavailable: 32-bit Fedora 29 changes

commit e9ba9783a9819536fb7c155984e17adf8aecb6d7
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Wed Oct 16 07:34:10 2019 +1100

    qa/check.callback.sample: refine ./870 output cmp
    
    Need to strip timestamps from output, then compare.

commit 34ec711ce5125b51c7cfd9c37e44b553eac3b161
Author: Ken McDonell <kenj@kenj.id.au>
Date:   Tue Oct 15 17:17:32 2019 +1100

    qa/admin/other-packages/manifest: updates for Arch Linux
